packages/forms/src/UIForm/UIForm.component.js

```diff
@@ -249,6 +250,15 @@ export class UIFormComponent extends React.Component {
 		return isValid;
 	}
 
+	onSubmitHover(event) {
+		const { properties } = this.props;
+		if (this.props.moz) {
```

Comment:
I don't think we need that anymore. It was introduced to make UIForm compatible with react-jsonschema-form lib api.
But since it's a new feature, let's stick to only 1 clean api.